index.html

```diff
@@ -45,6 +45,183 @@ <h1 class="headline">
                 </ul>
             </div>
         </section>
+        <main class="project-section">
+           
+            <div class="project">
+                <img src="./assets/images/project-1.png" alt="project-screenshot">
+                <h3 class="project-name">
+                    Tonic
+                </h3>
+                <div class="developer-info">
+                    <h4 class="developer-name">
+                        Ishpaul
+                    </h4>
+                    <ul>
+                        <li>● Back End Dev</li>
+                        <li>● 2022</li>
+                    </ul>
+                </div>
+                <p class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages-used">
+                    <li class="language">html</li>
+                    <li class="language">css</li>
+                    <li class="language">javascript</li>
+                </ul>
+                <button  type="button" class="project-link" >
+                    See Project
+                </button>
+            </div>
+
+            <div class="project">
+                <img src="./assets/images/project-2.png" alt="project-screenshot">
+                <h3 class="project-name">
+                    Multi-Post<br>Stories
+                </h3>
+                <div class="developer-info">
+                    <h4 class="developer-name">
+                        Ishpaul
+                    </h4>
+                    <ul>
+                        <li>● Back End Dev</li>
+                        <li>● 2022</li>
+                    </ul>
+                </div>
+                <p class="project-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages-used">
+                    <li class="language">html</li>
+                    <li class="language">css</li>
+                    <li class="language">javascript</li>
+                </ul>
+                <button  type="button" class="project-link">
+                    See Project
+                </button>
+            </div>
```

Comment:
- [ ] Optional.
Since the previous reviewer didn't report this as an issue, I still strongly recommend you fix this. 
Won't it look better if the image had equal margins on both left and right sides? What do you think? 
![card](https://user-images.githubusercontent.com/33268927/170675722-81070d62-3bb2-4505-a735-f87a00ac79ed.png)